apps/climatemappedafrica/src/lib/data/common/index.js

```diff
@@ -33,6 +33,24 @@ function getFooter(siteSettings) {
   };
 }
 
+function getMenus(siteSettings) {
+  const {
+    connect: { links = [] },
+    primaryNavigation: { menus = [], connect = [] },
+    primaryLogo,
+    drawerLogo,
```

Comment:
drawerLogo? Not sure why this is from the CMS. Is it something customisable? Otherwise this is just another icon (part of the theme).

Reply:
@kilemensi Yes. We need a different logo because the background of the drawer is blue, and the main logo would not look well
![image](https://github.com/user-attachments/assets/ea1b9192-93b8-47ae-84e8-728d08168cb5)
Sakina provided a new one that works very well as shown in the screenshots above

Reply:
Damn, that's one logo too many (not your fault)... lets add a story to track it but it shouldn't block this PR

Reply:
Tracking in: #961